firmware/controllers/trigger/trigger_decoder.cpp

```diff
@@ -65,7 +65,9 @@ float actualSynchGap;
 #endif /* ! EFI_PROD_CODE */
 
 #if ! EFI_UNIT_TEST || defined(__DOXYGEN__)
+#if EFI_TUNER_STUDIO || defined(__DOXYGEN__)
```

Comment:
I believe we should not be adding second condition here but replacing current "! EFI_UNIT_TEST" with EFI_TUNER_STUDIO, same below

Reply:
I've added some comments inside efifeatures.h on this matter.